src/app/search/search-filters/loadouts.tsx

```diff
@@ -8,30 +8,28 @@ export function loadoutToSearchString(loadout: Loadout | InGameLoadout) {
   return 'inloadout:' + quoteFilterString(loadout.name.toLowerCase());
 }
 
-// XXX: revisit when issue #9069 has been fixed and check if suggestions still fail
-//      see: https://github.com/DestinyItemManager/DIM/issues/9069
-//      to check: `inloadout:#hashta` should not suggest `inloadout:inloadout:#hashtag` (double prefix)
-//
-// function loadoutToSuggestions(loadout: Loadout) {
-//   return [
-//     quoteFilterString(loadout.name.toLowerCase()),
-//     ...getHashtagsFromNote(loadout.name),
-//     ...getHashtagsFromNote(loadout.notes),
-//   ].map(suggestion => 'inloadout:' + suggestion);
-// }
+// related: https://github.com/DestinyItemManager/DIM/issues/9069
+// sanity check: `inloadout:#hashta` should not suggest `inloadout:inloadout:#hashtag` (double prefix)
+function loadoutToSuggestions(loadout: Loadout) {
+  return [
+    quoteFilterString(loadout.name.toLowerCase()), // loadout name
+    ...getHashtagsFromNote(loadout.name), // #hashtags in the name
+    ...getHashtagsFromNote(loadout.notes), // #hashtags in the notes
+  ].map((suggestion) => 'inloadout:' + suggestion);
+}
 
 const loadoutFilters: FilterDefinition[] = [
   {
     keywords: 'inloadout',
     format: ['simple', 'freeform'],
 
-    suggestionsGenerator: ({ loadouts }) => loadouts?.map(loadoutToSearchString),
+    suggestionsGenerator: ({ loadouts }) => loadouts?.flatMap(loadoutToSuggestions),
 
     description: tl('Filter.InLoadout'),
-    filter: ({ filterValue, loadoutsByItem }) => {
+    filter: ({ lhs, filterValue, loadoutsByItem }) => {
       // the default search:
       // is:inloadout
-      if (filterValue === 'inloadout') {
+      if (lhs === 'is') {
```

Comment:
This change prevents `inloadout:inloadout` from having the same result as `is:inloadout`.